test/expr/postfix/call/construction.swift

```diff
@@ -113,7 +113,7 @@ protocol P2 {
 
 func constructExistentialValue(_ pm: P.Type) {
   _ = pm.init()
-  _ = P() // expected-error{{protocol type 'P' cannot be instantiated}}
+  _ = P() // expected-error{{instance member 'init' cannot be used on type 'P'}}
```

Comment:
Old diagnostic was better here, see my comment in `test/Constraints/protocols.swift`